app_to_web.md

```diff
@@ -76,6 +76,55 @@ For a site to enable App<->Web attribution after integrating with the Web API, t
 
 A reporting origin responding with the `Attribution-Reporting-Register-OS-Source` or `Attribution-Reporting-Register-OS-Trigger` headers while `Attribution-Reporting-Support` does not contain `os` will cause the entire registration to fail (including web registration).
 
+## Optional: debugging reports
+
+The Cross App and Web Attribution Measurement is a new and fairly complex way
+to do attribution measurements. A successful attribution requires both the
+browser and OS-level support. As such, we are open to introducing a mechanism to
+learn more information about OS registrations. This ensures that the API can be
+better understood and help flush out any bugs (either in browser or OS or caller
+code).
+
+The reporting origins may opt in to receiving debugging reports by adding a new
+boolean [parameter](https://httpwg.org/specs/rfc8941.html#param) `debug-reporting` to the
+[list](https://httpwg.org/specs/rfc8941.html#list)
+[item](https://httpwg.org/specs/rfc8941.html#item) in the
```

Comment:
IMO it's confusing to mention both list and item here. I'd just say `items`.

Reply:
Updated to just use items. Thanks.